### ISSUE TITLE
Incorrectly flagging line breaks in media rulesets as error

### ISSUE THREAD

**Issue by simonsmith** [suitcss/stylelint-config-suitcss#6]
This common pattern in many SUIT utils shows as an error:

``` css
@media (--sm-viewport) {
x <-- this line is an error
  .u-sm-afterNone {
    margin-left: 0 !important;
  }

}
```

I can fix it by setting:

``` js
    "rule-nested-empty-line-before": "always-multi-line"
```

@jeddy3 Before I change it just wanted to know if there is something I've missed as to why it had `except: ["first-nested"]`? Cheers

**Comment by jeddy3** [comment]
> @jeddy3 Before I change it just wanted to know if there is something I've missed as to why it had except: ["first-nested"]? 

I think that's a case of my personal preference sneaking. I don't think either style is mentioned in the styleguide, and so I went with my personal preference. In hindsight I should've checked the existing SUIT  components and utils, and gone with what I found in there.

So, feel free to remove the `except` option :) I think it'll need to be a major version bump.

**Comment by simonsmith** [comment]
Great, thanks. And yep, was going to major it :)

**Comment by simonsmith** [comment]
@jeddy3 One further question, with regard to `npm run release`. What steps do I need to follow before running that command?

**Comment by jeddy3** [comment]
First, you'll need to [generate a Github token](https://github.com/MoOx/npmpub#requirements). You'll only need to do this once.

Then, for each release, you'll need to add an entry to the changelog. Something along the lines of

```md
# Head

* Changed: the behaviour of the `rule-nested-empty-line-before` rule so that a empty line is always needed, irregardless of whether it's the first nested rule or not.
```

Once you're ready to release, you'll need to manually bump the version in the changelog and package.json, [like so](https://github.com/stylelint/stylelint/commit/6d0604675459a9ccabadcf573c50b43b295c12ce).

Finally, run `npm run release` - it'll generate a tag and release on github.. and then publish to npm (as well as running tests with clean dependencies and stuff like that).

Good luck :)

**Comment by simonsmith** [comment]
Excellent, thanks mate

**Comment by ntwb** [comment]
You'll have a PR to update to npmpub v3 in a couple of seconds :smile:

**Comment by jeddy3** [comment]
As an aside, thanks again to you both for taking ownership of these configs. It's very much appreciated :)

**Comment by simonsmith** [comment]
No worries, it makes perfect sense to do it this way :+1:

**Comment by simonsmith** [comment]
Fixed in 4.0
